Makefile

```diff
@@ -33,18 +33,20 @@ WINDOWS_IMAGE_TAGS=ltsc2019 1909 2004 20H2
 WINDOWS_BASE_IMAGES=$(BASE_IMAGE_LTSC2019) $(BASE_IMAGE_1909) $(BASE_IMAGE_2004) $(BASE_IMAGE_20H2)
 
 all: gce-pd-driver gce-pd-driver-windows
-gce-pd-driver:
+gce-pd-driver: require-GCE_PD_CSI_STAGING_VERSION
 	mkdir -p bin
 	go build -mod=vendor -gcflags=$(GCFLAGS) -ldflags "-X main.version=$(STAGINGVERSION)" -o bin/${DRIVERBINARY} ./cmd/gce-pd-csi-driver/
 
-gce-pd-driver-windows:
+gce-pd-driver-windows: require-GCE_PD_CSI_STAGING_VERSION
 	mkdir -p bin
 	GOOS=windows go build -mod=vendor -ldflags -X=main.version=$(STAGINGVERSION) -o bin/${DRIVERWINDOWSBINARY} ./cmd/gce-pd-csi-driver/
 
-build-container: require-GCE_PD_CSI_STAGING_IMAGE init-buildx
-	$(DOCKER) buildx build --platform=linux \
+build-container: require-GCE_PD_CSI_STAGING_IMAGE require-GCE_PD_CSI_STAGING_VERSION init-buildx
+	$(DOCKER) buildx build --platform=linux --progress=plain \
```

Comment:
what is progress here?

Reply:
It makes the output much cleaner on some terminal types. Otherwise docker gets very noisy.

Also on some platforms it seems that autodetecting the terminal works badly and can cause docker to crash :-(